test/e2e/framework/framework.go

```diff
@@ -128,15 +134,10 @@ func New(kubeConfigPath string) (*Framework, cleanUpFunc, error) {
 		return nil, nil, errors.Wrap(err, "creating scheduling v1 client failed")
 	}
 
-	openshiftConfigClient, err := openshiftconfigclientset.NewForConfig(config)
-	if err != nil {
-		return nil, nil, errors.Wrap(err, "creating openshift config v1 client failed")
-	}
-
 	f := &Framework{
 		RestConfig:               config,
 		OperatorClient:           operatorClient,
-		OpenShiftConfigClient:    openshiftConfigClient,
+		OpenshiftConfigClient:    openshiftConfigClient,
```

Comment:
This change seems to cause the test failure, @dgrisonnet mind having a look at this?